### PR TITLE
Add functions to list available blobs

### PR DESCRIPTION
**Pull request by katcipis**

### CODE REVIEW

File: azure/storage.sh [hunk 1]
```diff
@@ -215,6 +215,62 @@ fn azure_storage_container_exists_by_resgroup(containername, accountname, resgro
...
+	accountname,
+	resgroup,
+	numresults
+) {
+	accountkey, err <= _azure_storage_account_get_key_value($accountname, $resgroup)
+	if $err != "" {
+		return (), $err
+	}
+
+	res, err <= azure_storage_blob_list(
```
Comment (tiago4orion) [inline azure/storage.sh]: :(

Comment (katcipis) [reply azure/storage.sh]: elaborate the bad feeling a little better =P

Comment (tiago4orion) [reply azure/storage.sh]: your programming language is buggy

Comment (katcipis) [reply azure/storage.sh]: this is what you get for using stuff you find in the internet =/

**Pull request merged**